packages/renderer/src/lib/image/PushImageModal.svelte

```diff
@@ -108,7 +108,7 @@ $: window
         </p>{/if}
     </div>
 
-    <div class="h-[185px]" hidden={initTerminal === false}>
+    <div class="h-[185px]" hidden={initTerminal === false} role="term">
```

Comment:
question: why the role term would be on this one and not inside the TerminalWindow object which seems to be more legit to be the terminal ?

Reply:
moved to terminal component